<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupsWindow</class>
 <widget class="QDialog" name="GroupsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_addGroup">
       <property name="text">
        <string>Add group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_removeGroup">
       <property name="text">
        <string>Remove group</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_id">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_teacher">
         <property name="text">
          <string>Teacher: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_subject">
         <property name="text">
          <string>Subject:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_date">
         <property name="text">
          <string>Date of begin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_editGroup">
         <property name="text">
          <string>Edit group</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_addStudent">
       <property name="text">
        <string>Add student</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_removeStudent">
       <property name="text">
        <string>Remove student</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
